AutomatedTesting/Gem/PythonTests/scripting/TestSuite_Periodic.py

```diff
@@ -217,12 +212,26 @@ class TestScriptCanvasTests(object):
     """
     The following tests use hydra_test_utils.py to launch the editor and validate the results.
     """
+    def test_NodePalette_HappyPath_CanSelectNode(self, request, editor, launcher_platform):
+        expected_lines = [
+            "Success: Category can be selected",
+            "Success: Node can be selected"
+        ]
+        hydra.launch_and_validate_results(
+            request,
+            TEST_DIRECTORY,
+            editor,
+            "NodePalette_HappyPath_CanSelectNode.py",
+            expected_lines,
+            auto_test_mode=False,
+            timeout=60,
+        )
 
-    @pytest.mark.skip(reason="Test fails to find expected lines, it needs to be fixed.")
     def test_FileMenu_Default_NewAndOpen(self, request, editor, launcher_platform):
         expected_lines = [
-            "File->New action working as expected: True",
-            "File->Open action working as expected: True",
+            "Verified no tabs open: True",
+            "New tab opened successfully: True",
+            "Open file window triggered successfully: True"
```

Comment:
This will cause a collision in your other PR https://github.com/o3de/o3de/pull/9288 . This change should only be reflected in that PR and vice-versa.

Once one of these gets in, you'll have to fetch latest and merge the two files together and push the merged changes up.